inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.15/src/test/java/org/apache/inlong/sort/tests/utils/StarRocksManager.java

```diff
@@ -62,17 +62,20 @@ public static String getNewStarRocksImageName() {
     }
 
     public static void initializeStarRocksTable(StarRocksContainer STAR_ROCKS) {
+        initializeStarRocksTable(STAR_ROCKS, "id");
+    }
+    public static void initializeStarRocksTable(StarRocksContainer STAR_ROCKS, String id) {
```

Comment:
parameter id -> primaryKey , since id is a actual field name.

Reply:
fix it， thx～